modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -414,6 +416,14 @@ export class BitGoAPI implements BitGoBase {
         req.set('HMAC', requestProperties.hmac);
       }
 
+      if (this.getAdditionalHeadersCb) {
+        const data = serializeRequestData(req);
+        const additionalHeaders = this.getAdditionalHeadersCb(method, url, data);
+        for (const { key, value } of additionalHeaders) {
+          req.set(key, value);
+        }
+      }
```

Comment:
move this up a few lines into the same block where we check for token, and re-use the data prop so we don't have to serialize the payload twice